jekyll-llnl-theme.gemspec

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+
+Gem::Specification.new do |spec|
+  spec.name          = "jekyll-llnl-theme"
+  spec.version       = "0.1.0"
+  spec.authors       = ["Elsa Gonsiorowski", "Ian Lee", "Lawrence Livermore National Security, LLC"]
+  spec.email         = ["gonsie@me.com"]
```

Comment:
Do we want to use your work email?

Reply:
Good idea, but no... I use my personal email for most github-y things.